doc/src/pair_meam.rst

```diff
@@ -11,7 +11,9 @@ Syntax
 
 .. code-block:: LAMMPS
 
-   pair_style meam
+   pair_style meam [ms]
+
+* ms = use multi-state MEAM (optional)
```

Comment:
Using "ms" as a keyword is not a very user friendly choice. This should be more descriptive like "multistate".
An alternative (and my personal preference in this case) would be to have a "wrapper" pair style "msmeam". Where the derived class has no added functionality except for initializing the "msmeam" flag to 1
This has the advantage of the pair style name matching the published name of the method and retaining the property that MEAM pair style commands do not have arguments.

Reply:
@akohlmey Good point.  Perhaps we should follow the example of the PairEAM, which has many `eam/*` styles implemented as child classes.  The new command should be either `pair ms/meam` or `pair meam/ms`, I prefer the first one.

Reply:
Hmmm... if we want to be consistent with EAM, it should be meam/ms. I like msmeam a little better than ms/meam. With style names that has suffixes it usually the suffix and not the prefix that indicates the modification; msmeam avoids that.
@sjplimp would you mind being the tie-breaker here? :wink:

Reply:
I think meam/ms is consistent with other pair styles variants.  Also easier for users to understand and find
since pair style meam and meam/ms will be adjacent in the alphabetic listing.

Reply:
done